src/distributed-graphics-editions/DistributedGraphicsEdition.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.13;
 
-/**
```

Comment:
the tiniest of nits

Reply:
i can explain myself: this fixes foundry/forge autodoc feature which was useful. removing this makes the comment not get parsed by docs tools.